views/settings/lostinfo.tt

```diff
@@ -0,0 +1,74 @@
+[%# Request email for lost username or password reset.
+  #
+  # Authors:
+  #      import r26.1 livejournal -- original page
+  #      Jen Griffin <kareila@livejournal.com> -- TT conversion
+  #
+  # Copyright (c) 2008-2020 by Dreamwidth Studios, LLC.
+  #
+  # This program is free software; you may redistribute it and/or modify it under
+  # the same terms as Perl itself.  For a copy of the license, please reference
+  # 'perldoc perlartistic' or 'perldoc perlgpl'.
+%]
+
+[%- sections.title = '.title' | ml -%]
+[%- CALL dw.active_resource_group( "foundation" ) -%]
+
+[%- sections.head = BLOCK %]
+    <style type="text/css">
+        #content input, #content label { width: auto; display: inline; }
+    </style>
+[% END %]
+
+[%- IF captcha.enabled -%]
+<h2>[% 'captcha.title' | ml %]</h2>
+[%- END -%]
+
+<form action='lostinfo' method='post'>
+  [% dw.form_auth %]
+  [% captcha.print %]
+
+  <h2>[% '.lostpassword.title' | ml %]</h2>
+  <p>[% '.lostpassword.text' | ml( sitename = site.nameshort ) %]</p>
+
+  [% form.textbox( label = dw.ml( '.enter_username' ), id = 'userlost',
+                   name = 'user', size = 30, maxlength = username_maxlength,
+                   onkeyup = 'enable_pass();', onchange = 'enable_pass();' ) %]
+
+  <br />
+
+  [% form.textbox( label = dw.ml( '.enter_email_optional' ), name = 'email_p',
+                   size = 30, maxlength = 50 ) %]
+
+  <br />
+
+  [% form.submit( name = 'lostpass', id = 'lostpass',
+                  value = dw.ml( '.btn.proceed' ) ) %]
+
+  <h2>[% '.lostusername.title' | ml %]</h2>
+  <p>[% '.lostusername.text'   | ml %]</p>
+
+  [% form.textbox( label = dw.ml( '.enter_email' ), name = 'email_u',
+                   id = 'email_u', size = 30, maxlength = 50,
+                   onkeyup = 'enable_user();', onchange = 'enable_user();' ) %]
+
+  <br />
+
+  [% form.submit( name = 'lostuser', id = 'lostuser',
+                  value = dw.ml( '.btn.proceed' ) ) %]
+
+  <script type="text/javascript">
+  function enable_pass () {
+      document.getElementById("lostpass").disabled =
```

Comment:
👍🏼 

IDK if this came out clearly last time around, but this IS a one-to-one translation of the old JS. The old sixapart libraries implemented their own `$()` function that, even though it _looks_ like the jQuery thingy, really just does `getElementById` under the hood. Presumably this is part of why old 6alib pages don't play nice w/ jq.

I suppose we could also update these to jQuery calls, but then we'd also need to tweak the subsequent method calls (since jQuery doesn't return bare DOM elements like `getElementById` does) and would also need to split the JS out into separate files because mid-page jQuery is a no-no on Foundation pages. So this is the move unless you need to update the JS for other reasons.